### DISCUSSION TITLE
Icon Request: icon-scissors-comb

### DISCUSSION THREAD

**Discussion by viadom** [FortAwesome/Font-Awesome#21221] [Icon Requests]
Hi, is it possible to have an icon-scissors-comb like this one, but without the circle around? need it for my website in haircut business :) Thank you
![18087553-round-icon-of-hair-salon-with-scissors-and-comb-on-white-background](https://cloud.githubusercontent.com/assets/8556227/4043793/796eda68-2d17-11e4-8204-d906e7e07832.jpg)

**Comment by CoDEmanX** [comment]
There's already scissors, and here's a simple comb:
![comb](https://cloud.githubusercontent.com/assets/1309249/4043962/d77764b6-2d19-11e4-8de0-e02c216ff471.png)
https://dl.dropboxusercontent.com/u/39101475/comb.svg

**Comment by viadom** [comment]
yes but this is not one icon.

**Comment by CoDEmanX** [comment]
Does it matter?
You can use both, e.g. http://jsfiddle.net/8agsgaac/

Or use InkScape to combine both into a single icon, then go to https://icomoon.io/app/ and create a FontAwesome superset.

**Comment by viadom** [comment]
I use Iconbox and I can't use more than one Icon for one box.
here you can see what I mean
![icon box](https://cloud.githubusercontent.com/assets/8556227/4044214/91a2bc2a-2d1d-11e4-88ab-22f2812f9d12.JPG)
I'll try what you say to combine both of them

**Comment by tagliala** [comment]
@CoDEmanX

This is a very specific use case so your suggestion of a custom build makes sense, but this request is legit and provides all infos requested (proper name, use case, example)

**Comment by viadom** [comment]
ok I tried to use icomoon and InkScape but i'm stuck ! 
I made a svg but I cannot load it to use it in iconbox

**Comment by CoDEmanX** [comment]
Here's an example using a custom font created with IcoMoon:
http://jsfiddle.net/3r98mxjk/

I included the link for the CSS + fonts. I had turn the contour into a path to make it work. I also combines, union'd and simplified the comb, so that there aren't unnecessary paths and nodes.

Note the different icon names with IcoMoon fonts: they are prefixed `icon-`, not `fa-`.

My icon names are `icon-comb` and `icon-hairdresser` (0xff01 and 0xff02).

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
